test_suite/conftest.py

```diff
@@ -1,19 +1,46 @@
 import json
+
+import re
 import time
+import logging
 import pytest
-import re
-import os
 import requests
-
-from requests.adapters import HTTPAdapter
-from requests.packages.urllib3.util.retry import Retry
+import os
+import configparser
 
 from py.xml import html
 from pytest_html import extras
+from requests.adapters import HTTPAdapter
+from requests.packages.urllib3.util.retry import Retry
+from reportportal_client import RPLogger
 
 from lib import test_lib
 
 
+@pytest.fixture(scope="session")
+def rp_logger():
+    logger = logging.getLogger(__name__)
+    logger.setLevel(logging.DEBUG)
+    logging.setLoggerClass(RPLogger)
+    return logger
+
+
+def get_rp_config():
```

Comment:
```suggestion
def set_report_portal_config():
```
I suggest renaming this function to `set_report_portal_config`
(primarily change get by set since it's actually setting the config)